configuration.rst

```diff
@@ -175,6 +176,27 @@ server-unix-socket-mode
 
     server-unix-socket-mode = "660"
 
+.. _openapi-mode:
+
+openapi-mode
+------------
+
+  Specifies how the OpenAPI output should be displayed:
+
+  .. code:: bash
+
+    # Follows the access control for the JWT role.
+    # Shows information depending on the permissions that the user making the request has
+    openapi-mode = "follow-privileges"
+
+    # Ignores the access control for the JWT role.
+    # Shows all the exposed information, regardless of the permissions the user making the request has
```

Comment:
Ditto. Also, "privileges" is more precise than "access control"([doc](https://www.postgresql.org/docs/13/ddl-priv.html#PRIVILEGE-ABBREVS-TABLE))
```suggestion
    # Ignores the privileges of the JWT role claim
    # Shows all the exposed information, regardless of the permissions the role making the request has
```